### ISSUE TITLE
Setup with Google authentication?

### ISSUE THREAD

**Issue by stevenhill1** [signalnerve/roam-backup#12]
I am trying to set up, and hit an issue. When I started using Roam I used the option to use google to authenticate, so I don't have anything to include in roamPassword. Is there anyway round this?

**Comment by Stvad** [comment]
yes, you can reset password and get an explicit password that way

**Comment by kvnandula04** [comment]
> yes, you can reset password and get an explicit password that way

Thanks for that!
